tests/test_distro.py

```diff
@@ -741,7 +736,7 @@ def _test_outcome(
             DISTROS_DIR,
             distro_name + version,
             "etc",
-            "{0}-{1}".format(release_file_id, release_file_suffix),
+            "{}-{}".format(release_file_id, release_file_suffix),
```

Comment:
What about an F-string here too @jdufresne ?

```suggestion
            f"{release_file_id}-{release_file_suffix}",
```

Reply:
Thanks for the suggestion. I've converted this to an f-string in the latest revision. I also converted a few other discovered by the tool [flynt](https://github.com/ikamensh/flynt).